### PR TITLE
add vue-i18n package, and show example usage via 3 strings in user

### PR DESCRIPTION
**Pull request by niniane**
profile

### CODE REVIEW

File: src/plugins/i18n.ts [hunk 1]
```diff
@@ -0,0 +1,24 @@
+import Vue from 'vue';
```
Comment (niniane) [inline src/plugins/i18n.ts]: Put copyright.

File: src/plugins/i18n.ts [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file. 
+
+import Vue from 'vue';
+import VueI18n from 'vue-i18n';
+
+Vue.use(VueI18n);
+
+const messages = {
+  'en': {
```
Comment (derat) [inline src/plugins/i18n.ts]: optional for later: omit quotes on object properties

File: src/plugins/i18n.ts [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+    team: 'Team',
+  }, 
+  'es': {
+    yourName: 'Tu Nombre',
+    individual: 'Individual',
+    team: 'Grupo',
+  }
+};
+
+export const i18n = new VueI18n({
```
Comment (derat) [inline src/plugins/i18n.ts]: if this is going to be the only export, should we just do something like `export default new VueI18n(...`?

Comment (derat) [reply src/plugins/i18n.ts]: (i might be getting the syntax wrong)

File: src/plugins/i18n.ts (hunk 2 above)
Comment (niniane) [reply src/plugins/i18n.ts]: Ok, omitted quotes.

File: src/plugins/i18n.ts (hunk 3 above)
Comment (niniane) [reply src/plugins/i18n.ts]: Ok, I used export default.

**Pull request merged**